Mathlib/Analysis/NormedSpace/TrivSqZeroExt.lean

```diff
@@ -176,4 +176,90 @@ end Field
 
 end Topology
 
+/-!
+### The $ℓ^1$ norm on the trivial square zero extension
+
+This is not particularly canonical, but is sufficient to obtain a normed algebra, and thus
+to use `exp_add_of_commute`.
```

Comment:
There may be some argument that it is canonical. This isn't it, but related: for regular normed algebras, the ℓ¹-norm on the unitization is maximal with respect to all norms (making it into a `NormedAlgebra` satisfying `NormOneClass`) for which the embedding is an isometry. It's possible there is something similar for `TrivSqZeroExt`, but it's also possible they are not that closely related.